CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+# Changelog
+
+<!-- 
+Generate the changelog using the lerna-changelog tool. 
+Run the following command to generate changelog content for pull requests that were merged since the time when the last git tag was created.
```

Comment:
`pull requests that have been merged since` (since -> present perfect) and if you say `Run the following command` I guess it would be better to add the sentence with the `:` rather than `.`?